app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java

```diff
@@ -102,4 +104,19 @@ public void onButtonSumPressed(){
     public void onButtonSubtractPressed(){
         presenter.onOperatorPressed(MINUS);
     }
+
+    @OnClick(R.id.button_multiply)
+    public void onButtonMultiplyPressed(){
+        presenter.onOperatorPressed(MULTIPLICATION);
+    }
+
+    @OnClick(R.id.button_clear)
+    public void onButtonClearPressed(){
+        presenter.onClearPressed();
+    }
+
+    @OnLongClick(R.id.button_clear)
+    public void onButtonClearLongClicked(){
```

Comment:
remove this as you do not use it any more

Reply:
done